src/components/Articles/SingleArticles/ArticlesById.jsx

```diff
@@ -0,0 +1,42 @@
+import { getArticlesById } from '../../../utils';
+import { useEffect, useState } from 'react';
+import { useParams } from'react-router-dom';
+import './ArticlesById.css'
+
+
+
+export default function ArticlesById() {
+
+  const { article_id } = useParams();
+  const [articleById, setArticleById] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+
+    useEffect(() => { 
+      getArticlesById(article_id).then(response => {
+        setArticleById(response.data.article);
+        console.log(response.data.article);
+        setLoading(false);
+      });
+    }, []);
+
+
+
+  return (
+    <>
+
+      <h1 className='title'>{articleById.title}</h1>
+      <p>Author: {articleById.author}</p>
+      <p>Topic: {articleById.topic}</p>
+      
+
+      <div className='img-and-text'>
+        <p className='body'>{articleById.body}</p>
+        <img class='img' src={articleById.article_img_url}></img>
+      </div>
+      <p>Created: {articleById.created_at}</p>
```

Comment:
Would be better to have the timestamp formatted in a bit more of a human readable way, than the default. Have a think how you could manipulate the data yourself to do this, or look into some packages that can do it for you 💡